app/experimenter/experiments/forms.py

```diff
@@ -566,48 +566,68 @@ class RadioWidget(forms.widgets.RadioSelect):
 class ExperimentRisksForm(ChangeLogMixin, forms.ModelForm):
     RADIO_OPTIONS = ((False, "No"), (True, "Yes"))
 
+    def coerce_truthy(value):
+        if value.lower() == "true":
+            return True
+        elif value.lower() == "false":
+            return False
+
     # Radio Buttons
-    risk_internal_only = forms.ChoiceField(
+    risk_internal_only = forms.TypedChoiceField(
         label=Experiment.RISK_INTERNAL_ONLY_LABEL,
         help_text=Experiment.RISK_INTERNAL_ONLY_HELP_TEXT,
         choices=RADIO_OPTIONS,
         widget=RadioWidget,
+        coerce=coerce_truthy,
```

Comment:
I think you can just say `coerce=bool` here?

Reply:
The value getting passed to these is a string of either `"True"` or `"False"` which come from the `RADIO_OPTIONS` after going through a string conversion in Django-land. And `bool("False") => True` which isn't what we want.